helper/testhelpers/mongodb/mongodbhelper.go

```diff
@@ -44,20 +40,14 @@ func PrepareTestContainerWithDatabase(t *testing.T, version, dbName string) (fun
 		if dbName != "" {
 			connURL = fmt.Sprintf("%s/%s", connURL, dbName)
 		}
-		dialInfo, err := ParseMongoURL(connURL)
-		if err != nil {
-			return nil, err
-		}
 
-		session, err := mgo.DialWithInfo(dialInfo)
+		ctx, _ = context.WithTimeout(context.Background(), 10*time.Second)
+		client, err := mongo.Connect(ctx, options.Client().ApplyURI(connURL))
 		if err != nil {
 			return nil, err
 		}
-		defer session.Close()
 
-		session.SetSyncTimeout(1 * time.Minute)
-		session.SetSocketTimeout(1 * time.Minute)
-		err = session.Ping()
+		err = client.Ping(ctx, readpref.Primary())
 		if err != nil {
 			return nil, err
 		}
```

Comment:
I'm thinking we should disconnect the client after the `Ping()` call similar to what's shown in https://github.com/mongodb/mongo-go-driver#usage (except `t.Fatalf()` instead a `panic()`).

Reply:
Good point 👍 . Thank you!